src/tests/ftest/launch.py

```diff
@@ -106,8 +106,14 @@ def get_build_environment():
     build_vars_file = os.path.join(
         os.path.dirname(os.path.realpath(__file__)),
         "../../.build_vars.json")
-    with open(build_vars_file) as vars_file:
-        return json.load(vars_file)
+    try:
+        with open(build_vars_file) as vars_file:
+            return json.load(vars_file)
+    except ValueError:
+        return json.loads('{{"PREFIX": "{}"}}'.format(os.getcwd()))
+    except IOError as error:
+        if error.errno == errno.ENOENT:
+            return json.loads('{{"PREFIX": "{}"}}'.format(os.getcwd()))
```

Comment:
Should these alternatives only be acceptable in certain circumstances?  In other words is there a situation where if `../../.build_vars.json` does not exist the correct action would be to report an error?

If not would it make sense to bypass all of this work and just use `os.getcwd()` to define the prefix?  Possibly something to address later on.

Reply:
> Should these alternatives only be acceptable in certain circumstances?

Such as `args.list == true`?

> If not would it make sense to bypass all of this work and just use `os.getcwd()` to define the prefix?

I figured if the file does happen to exist, might as well use it.

Reply:
Done